packages/conf-git/conf-git.1.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://git-scm.com"
+authors: "Linus Torvalds"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-2.0-or-later"
+build: [["which" "git"]]
```

Comment:
What about
```suggestion
build: [["git" "--version"]]
```
completely dropping the depends field?
This way this can be compatible with windows